pom.xml

```diff
@@ -53,6 +53,26 @@
       <artifactId>junit</artifactId>
       <version>4.12</version>
     </dependency>
+
+    <dependency>
+        <groupId>com.squareup.retrofit2</groupId>
+        <artifactId>parent</artifactId>
+        <version>2.8.1</version>
+        <type>pom</type>
+    </dependency>
+
+    <dependency>
+        <groupId>com.squareup.retrofit2</groupId>
+        <artifactId>converter-gson</artifactId>
+        <version>2.9.0</version>
+    </dependency>
```

Comment:
These (as well as the gradle files and the Settings file) can be remove, I think

Reply:
I still see Retrofit, did you push your changes by any changes?

Could you also remove the `.gradle/` directory?